routes/index.js

```diff
@@ -22,18 +32,34 @@ router.get('/', function (req, res, next) {
 });
 
 router.post('/', upload.single('file'), function (req, res, next) {
-  var categoryId = 'Category ID ';
-  var category = 'Category';
   var workbook = xlsx.readFile(req.file.path);
   bluebird.map(workbook.SheetNames, function (item) {
     var jsonArray = xlsx.utils.sheet_to_json(workbook.Sheets[item]);
     return bluebird.map(jsonArray, function (obj) {
 
-      if (obj !== undefined && obj[categoryId] !== undefined && obj[category] !== undefined) {
-        var filePath = 'JSON_Files/' + obj[category].trim().replace(/\s>\s/g, '/');
-        var jsonFilePath = filePath + '/' + obj[categoryId] + '.json';
-        delete obj[categoryId];
-        delete obj[category];
+      if (obj !== undefined && obj['Category ID '] !== undefined && obj.Category !== undefined) {
+
+        obj.CategoryID = obj['Category ID '];
+        delete obj['Category ID '];
+        obj.MetaTitle = obj['Meta Title'];
+        delete obj['Meta Title'];
+        obj.MetaDescription = obj['Meta Description'];
+        delete obj['Meta Description'];
+        var newJsonData = {
+          documentName: req.file.originalname,
+          sheetName: item,
+          jsonObject: obj
+        };
+        dbServices.create(jsonDataModel, newJsonData);
+
+        obj['Meta Title'] = obj.MetaTitle;
+        delete obj.MetaTitle;
+        obj['Meta Description'] = obj.MetaDescription;
+        delete obj.MetaDescription;
+        var filePath = 'JSON_Files/' + obj.Category.trim().replace(/\s>\s/g, '/');
+        var jsonFilePath = filePath + '/' + obj.CategoryID + '.json';
+        delete obj.CategoryID;
+        delete obj.category;
```

Comment:
This logic can be implemented in with lesser number of lines. Also, please verify if this is generic. Don't think it is. We had mentioned that specifically during the meeting.